lightning/src/ln/peer_handler.rs

```diff
@@ -459,6 +459,17 @@ impl<Descriptor: SocketDescriptor, CM: Deref, L: Deref> PeerManager<Descriptor,
 		}
 	}
 
+	/// Append a message to a peer's pending outbound/write buffer, and update the map of peers needing sends accordingly
```

Comment:
"and insert its descriptor in the map of peers needing sends" ?

Reply:
I'm open to either phrasing. I believe you and Matt have the power to edit this though, so feel free to update it to whichever you prefer.